ls1intum/Artemis
# `Development`: Add latest version to database for better migration management in the future

### Checklist
#### General
#### Server

### Motivation and Context
We want to verify migration paths in the future. Therefore, we need to know which "latest" version was executed before the current "new" version is started

### Description
Store the latest version in the database and store it with native SQL statements

Note: I know the code quality is not the highest level, but it should be fine for now (except you find some really problematic aspects)

### Steps for testing
Start Artemis locally in this branch and check if a new database table with the latest version is created.
No SQL errors or startup errors should occur in the logs

### Review Progress





#### Code Review
#### Manual Tests
